bin/compile

```diff
@@ -80,7 +80,7 @@ info "node_modules cached: $modules_cached"
 
 echo ""
 
-info "NPM_CONFIG_PRODUCTION=$NPM_CONFIG_PRODUCTION"
```

Comment:
Hi @hunterloftis,
I can't seem to set the `NPM_CONFIG_PRODUCTION` variable to false by passing in 'NPM_CONFIG_PRODUCTION=false'. Did this change disable the ability to set the var?